### DISCUSSION TITLE
Round 21: Why actix plaintext values are different in Composite scores than in Plaintext results?

### DISCUSSION THREAD

**Discussion by Domram** [TechEmpower/FrameworkBenchmarks#7843] [General]
actix is displayed as the 5th best framework by Composite score, but according to that same Composite scores data, actix got 7,017,232 performance in the Plaintext test while if I look into the plaintext data I see that actix got much less than that, i.e. 3,661,817 .

Is this a mistake or am I not interpreting the data in the right way?
![Round21-Composite-1to5](https://user-images.githubusercontent.com/41471907/210800294-e4d451d0-ae01-4218-8a89-fd50e6fa90fb.png)
Plaintext:
![Round21-Plaintext-actix](https://user-images.githubusercontent.com/41471907/210800301-7c36136e-bb41-4400-8854-3098c83d6aef.png)

**Comment by fakeshadow** [comment]
https://github.com/TechEmpower/FrameworkBenchmarks/issues/7475#issuecomment-1190855363 and https://github.com/TechEmpower/FrameworkBenchmarks/issues/7475#issuecomment-1191598733 explained it.
